src/ophyd_async/epics/_backend/_p4p.py

```diff
@@ -122,20 +200,17 @@ def value(self, value):
         return self.choices[value["value"]["index"]]
 
     def get_datakey(self, source: str, value) -> DataKey:
-        return {
-            "source": source,
-            "dtype": "string",
-            "shape": [],
-            "choices": list(self.choices),
-        }
+        return _data_key_from_value(
+            source, value, choices=list(self.choices), dtype="string"
+        )
 
 
-class PvaEnumBoolConverter(PvaConverter):
+class PvaBoolConverter(PvaConverter):
```

Comment:
I preferred the original name, it converts a PvaEnum to a bool...

Reply:
I have changed it back.